releasenotes/notes/add_check_from_sparse_list-97f13fde87c7bcb6.yaml

```
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    Added a check process and a flag which is used to check if the qubit designated from input Pauli list is already assigned
+    to :func:`from_sparse_list` in the :class:`SparsePauliOp`. The flag is used if check is to be run. The default value is True.
+    See `#7916 <https://github.com/Qiskit/qiskit-terra/issues/7916>`__.
```

This should link the method directly 🙂 
```suggestion
    Allow checking for duplicate qubit indices in :meth:`~qiskit.quantum_info.SparsePauliOp.from_sparse_list`.
    The checks can be turned off by setting the new keyword argument ``do_checks`` to ``False`` 
    (it  is ``True`` per default).
```